src/sentry/tasks/post_process.py

```diff
@@ -120,6 +122,24 @@ def _capture_group_stats(job: PostProcessJob) -> None:
     metrics.incr("events.unique", tags=tags, skip_internal=False)
 
 
+def should_issue_owners_ratelimit(
+    cache_key,
+):
+    data = cache.get(cache_key)
+    if data is None:
+        requests = 1
+        window_start = datetime.now()
+        cache.set(cache_key, (requests, window_start), 60)
+    else:
+        requests = data[0] + 1
+        window_start = data[1]
+        # timeout should never be less than 0
+        timeout = max(60 - (datetime.now() - window_start).total_seconds(), 0)
```

Comment:
If I understand correctly, we would not cache (a timeout of 0) or cache with a timeout of 1 to 59 seconds.